packages/polaris-viz/src/components/LineChartRelational/components/CustomLegend/CustomLegend.tsx

```diff
@@ -1,6 +1,18 @@
+/* eslint-disable @shopify/strict-component-boundaries */
```

Comment:
I had to add this comment, otherwise I get an eslint error when importing:
import {HiddenLegendTooltip} from '../../../../LegendContainer/components/HiddenLegendTooltip';